### DISCUSSION TITLE
Quiz app ~Lohith

### DISCUSSION THREAD

**Discussion by Loafiee** [sst-inc/help#32] [Q&A - iOS] [answered]
So when I try to play again using the unwind segue, it doesn't go back to the start, doesn't update the questions, and doesn't restart the points system. 
<img width="641" alt="Screenshot 2021-08-23 at 4 20 18 PM" src="https://user-images.githubusercontent.com/87929393/130414771-f65d200f-bb46-47e7-9cb2-bf475321787c.png">
<img width="685" alt="Screenshot 2021-08-23 at 4 20 31 PM" src="https://user-images.githubusercontent.com/87929393/130414788-651b4fcf-173d-4d8d-bdf0-681853c10113.png">

https://user-images.githubusercontent.com/87929393/130414791-721edff1-235c-47dc-8952-2e8c37f133d0.mov

**Comment by Joeispro99** [comment, marked as answer]
Ok instead of putting your IBAction func goBack in your results view controller, do it in the main view controller. Then redo the steps after deleting the old unwind segue. Does it work?

**Reply by Ethan-Chew** [reply]
Please mark this answer as 'answer' if it works for you. Thanks!
